.circleci/config.yml

```diff
@@ -46,22 +47,11 @@ jobs:
       - checkout:
           path: ~/galoy-mobile
 
-      - run: 
-          name: Install Node 18
-          command: |
-            sudo apt-get update
-            sudo apt-get install -y ca-certificates curl gnupg
-            sudo mkdir -p /etc/apt/keyrings
-            curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | sudo gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
-
-            NODE_MAJOR=18
-            echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_$NODE_MAJOR.x nodistro main" | sudo tee /etc/apt/sources.list.d/nodesource.list
-
-            sudo apt-get update
-            sudo apt-get install nodejs -y
+      - node/install:
+          install-yarn: true
+          node-version: '18.18.2'
```

Comment:
We should use node 20?

Any reason why the 18 here?

Reply:
I was using Node 18 for development locally, does it work with Node 20?